src/shared/sam/cli/samCliContext.ts

```diff
@@ -55,26 +55,6 @@ export async function getSamCliVersion(context: SamCliContext): Promise<string>
     return result.versionValidation!.version!
```

Comment:
CI failing because we just need to delete some imports:

```
src/shared/sam/cli/samCliContext.ts(6,1): error TS6133: 'semver' is declared but its value is never read.
src/shared/sam/cli/samCliContext.ts(14,1): error TS6133: 'getLogger' is declared but its value is never read.

Reply:
Fixed